api/issue.go

```diff
@@ -149,9 +149,9 @@ type UpdateSchemaDetail struct {
 type UpdateSchemaContext struct {
 	// MigrationType is the type of a migration.
 	MigrationType db.MigrationType `json:"migrationType"`
-	// UpdateSchemaDetail is the details of schema update.
+	// DetailList is the details of schema update.
```

Comment:
Not sure which one is better. I'd not touch it unless necessary.

Reply:
It's different from `UpdateSchemaGhostContext`.

IMO, we should prevent unnecessary verbosity, such as `UpdateSchemaXxx.Yyy` instead of `UpdateSchemaXxx.UpdateSchemaYyy`.

Reply:
It's subjective and I don't have a preference here. It's recommended to not touch existing code unless necessary in Software Engineering. (you never know :-) )

Reply:
SG.
Anyway, maybe we still want consistency between `UpdateSchemaContext` and `UpdateSchemaGhostContext`?

Reply:
Yes, consistency + simplicity wins.